test-sbt/jvm/src/test/scala/zio/test/sbt/ZTestFrameworkSpec.scala

```diff
@@ -18,64 +20,76 @@ object ZTestFrameworkSpec {
 
   def tests: Seq[Try[Unit]] = Seq(
     test("should return correct fingerprints")(testFingerprints()),
-    test("should report events")(testReportEvents()),
-    test("should report durations")(testReportDurations()),
-    test("should log messages")(testLogMessages()),
-    test("should correctly display colorized output for multi-line strings")(testColored()),
-    test("should test only selected test")(testTestSelection()),
-    test("should return summary when done")(testSummary()),
-    test("should use a shared layer without re-initializing it")(testSharedLayer()),
-    test("should warn when no tests are executed")(testNoTestsExecutedWarning())
+    // TODO restore once we are calculating durations again
+    //test("should report durations")(testReportDurations()),
```

Comment:
Could also be part of issue to track follow up items.

Reply:
Added new issue number here